### PR TITLE
Misc doc fixes

### PR DESCRIPTION
**Pull request by bpradipt**
Signed-off-by: Pradipta Banerjee <pradipta.banerjee@gmail.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/170#" title="Author self-approved">bpradipt</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: docs/ocp_prereqs_powervs.md [hunk 1]
```diff
@@ -81,31 +112,36 @@ Further, the image disk should be minimum of 120 GB in size.
...
-- You can also use the following [python script](https://github.com/ocp-power-automation/infra/raw/master/scripts/images/convert_qcow2_ova.py) to convert Qcow2 image to OVA
-  - RHEL 8.2 Qcow2 image is available from the following [link](https://access.redhat.com/downloads/content/279/ver=/rhel---8/8.2/ppc64le/product-software)
-  - RHCOS Qcow2 image is available from the following [link](https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.5/)
+- You can also use the following [tool](https://github.com/ppc64le-cloud/pvsadm) to convert Qcow2 image to OVA.
+- Qcow2 Image Links
+  - RHEL 8.3 Qcow2 image is available from the following [link](https://access.redhat.com/downloads/content/279/ver=/rhel---8/8.3/ppc64le/product-software)
+  - CentOS 8.3 Wcow2 image is available from the following [link](https://cloud.centos.org/centos/8/ppc64le/images/CentOS-8-GenericCloud-8.3.2011-20201204.2.ppc64le.qcow2)
+  - RHCOS Qcow2 image is available from the following [link](https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rhcos/4.6/latest/rhcos-4.6.1-ppc64le-openstack.ppc64le.qcow2.gz)
```
Comment (yussufsh) [inline docs/ocp_prereqs_powervs.md:119]: May be we should mention here to download release specific RHCOS image.

File: docs/quickstart.md [hunk 2]
```diff
@@ -200,12 +200,12 @@ $ scp -r -i data/id_rsa root@158.175.161.118:~/openstack-upi/auth/\* .
 
 OpenShift CLI `oc` can be downloaded from the following links. Use the one specific to your client system architecture.
 
-- [Mac OSX](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/4.5/macosx/oc.tar.gz)
-- [Linux (x86_64)](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/4.5/linux/oc.tar.gz)
-- [Windows](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/4.5/windows/oc.zip)
+- [Mac OSX](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/latest/macosx/oc.tar.gz)
```
Comment (yussufsh) [inline docs/quickstart.md:203]: Just mentioning here again.. these oc links are deprecated.

File: docs/ocp_prereqs_powervs.md (hunk 1 above)
Comment (bpradipt) [reply docs/ocp_prereqs_powervs.md:119]: Yeah no harm

File: docs/quickstart.md (hunk 2 above)
Comment (bpradipt) [reply docs/quickstart.md:203]: Yes, thanks for pointing out. I'll update it

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/170#" title="Author self-approved">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/170#pullrequestreview-551062750" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by Prajyot-Parab — APPROVED** [review]
/approve

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/170#" title="Author self-approved">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/170#pullrequestreview-551145544" title="Approved">Prajyot-Parab</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/170#pullrequestreview-551062750" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [Prajyot-Parab,bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**